docs/src/_reference/command-line-interface.md

```diff
@@ -729,15 +736,24 @@ meltano job list <job_name> --format=json
 meltano job remove <job_name>
 ```
 
-### Examples
+##### Tasks
+
+A task should be of the same format as arguments supplied to [the `meltano run` command](#run), which can be any valid sequence of extractors, mappers, loaders, and plugin commands.
+Note that such a sequence is only valid if it is one of:
 
-##### Tasks and Blocks
+1. An extractor followed directly by a loader. E.g. `tap-gitlab target-postgres`
+1. An extractor followed by one or more mappers and then a loader. E.g. `tap-gitlab hide-gitlab-secrets target-postgres`
+1. A plugin command invocation. E.g. `dbt-postgres:run`
```

Comment:
@cjohnhanson I think it can just reference a full plugin as well if a user wants. It's just unlikely that they'll simple invoke the plugin as-is without a command / CLI args, right? @pandemicsyn can you validate?

Reply:
@tayloramurphy no today we require a command to run (would be a super easy change to remove that though, would just need to remove this check https://github.com/meltano/meltano/blob/dfd9650170ff97fd69fdff7ac8dc7dc0841bf663/src/meltano/core/block/singer.py#L97).

Reply:
@tayloramurphy @DouweM do we want to go ahead and allow runs as part of the scope of this issue/PR or make a follow-issue to enable?

Reply:
@cjohnhanson definitely a follow-up issue! I was just making sure we documented it correctly for the current state.

Reply:
I made https://github.com/meltano/meltano/issues/6188 to track